seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/splitter/AbstractJdbcSourceChunkSplitter.java

```diff
@@ -379,12 +380,38 @@ protected SnapshotSplit createSnapshotSplit(
     protected Column getSplitColumn(
             JdbcConnection jdbc, JdbcDataSourceDialect dialect, TableId tableId)
             throws SQLException {
-        Optional<PrimaryKey> primaryKey = dialect.getPrimaryKey(jdbc, tableId);
         Column splitColumn = null;
+        Table table = dialect.queryTableSchema(jdbc, tableId).getTable();
+
+        // first , compare user defined split column is in the primary key or unique key
+        Properties splitColumnProperties = new Properties();
```

Comment:
Why not just use Map?

Reply:
ping @XenosK

Reply:
Please use Map, properties are used because they interact with debezium, I think we should use Map in here just like other connectors.

Reply:
> Please use Map, properties are used because they interact with debezium, I think we should use Map in here just like other connectors.

ok